content/integrations/integrating-npm-with-external-services/about-access-tokens.mdx

```diff
@@ -25,13 +22,19 @@ You can work with tokens from the web or the CLI, whichever is easiest. What you
 npm token commands let you:
 
 - View tokens for easier tracking and management
-- Create new legacy tokens
+- Create new legacy tokens (deprecated)
 - Limit access according to IP address ranges (CIDR)
 - Delete/revoke tokens
 
 For more information on creating and viewing access tokens on the web and CLI, see "[Creating and viewing access tokens][create-token]".
 
-## About legacy tokens
+## About legacy tokens (Deprecated)
+
+<Note>
+
+**Warning:** Legacy access tokens are deprecated as of November 2025.
+
+</Note>
```

Comment:
The blue "Warning: xxx" block of text isn't ideal - not visible enough. Ideally we want a 

I am not sure if this would work or not (given https://github.com/npm/documentation/blob/5e0b688ce32037fae48b7c92731bba99acd331d8/src/mdx/components.js#L276-L279) but worth a try:
```suggestion
<Note variant="danger">

**Warning:** Legacy access tokens are deprecated as of November 2025.

</Note>
```